### PR TITLE
[MI-2435] Updated documetation to add the info about auto-connect feature

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ This plugin communicates with Microsoft Exchange Server through a companion serv
   
  You may use the `Regenerate` button to generate a new key. Ensure that the key is configured in the mattermost-plugin-exchange-ews-proxy's `.env` file so that the proxy can authenticate API requests made by this plugin.
 
+  - **Auto-Connect Users**: When set to 'true', all the users on Mattermost are automatically connected to Exchange via EWS proxy server.
```
Comment (ayusht2810) [inline README.md:55]:
```suggestion
  - **Auto-Connect Users**: When set to `true`, all the users on Mattermost are automatically connected to Exchange via EWS proxy server.
```

File: README.md [hunk 2]
```diff
@@ -52,6 +52,10 @@ This plugin communicates with Microsoft Exchange Server through a companion serv
...
+
+    **Note:** If an admin changes this setting from `false` -> `true`, then he/she has to disable and re-enable the plugin to auto-connect all the logged-in users. If he/she does not do so, then only the users who log in to Mattermost after the setting was enabled will auto-connect.
```
Comment (ayusht2810) [inline README.md:57]:
```suggestion
    **Note:** If an admin changes this setting from `false` -> `true`, then the user has to re-enable the plugin to auto-connect all the logged-in users. If the user does not do so, then only the users who log in to Mattermost after the setting was enabled will auto-connect.
```

**Review by ayusht2810 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -52,6 +52,10 @@ This plugin communicates with Microsoft Exchange Server through a companion serv
   
  You may use the `Regenerate` button to generate a new key. Ensure that the key is configured in the mattermost-plugin-exchange-ews-proxy's `.env` file so that the proxy can authenticate API requests made by this plugin.
 
+  - **Auto-Connect Users**: When set to `true`, all the users on Mattermost are automatically connected to Exchange via EWS proxy server.
```
Comment (Kshitij-Katiyar) [inline README.md:55]: Dont think there is a need for EWS proxy server here.
```suggestion
  - **Auto-Connect Users**: When set to `true`, all the users on Mattermost will be automatically connected to Exchange Calendar.
```

File: README.md [hunk 4]
```diff
@@ -52,6 +52,10 @@ This plugin communicates with Microsoft Exchange Server through a companion serv
...
+
+    **Note:** If an admin changes this setting from `false` -> `true`, then admin has to disable and re-enable the plugin to auto-connect all the logged-in users. If the admin does not do so, then only the users who log in to Mattermost after the setting was enabled will auto-connect.
```
Comment (Kshitij-Katiyar) [inline README.md:57]: We can frame this better as just an example don't copy this.

By updating the value from false to true only the users who log in to Mattermost after the setting was enabled will auto-connect. To auto-connect all the logged-in users the admin has to disable and re-enable the plugin.

**Review by Kshitij-Katiyar — APPROVED** [review]

**Pull request merged**